arch/arm64/boot/dts/qcom/sdm625-xiaomi-mido.dts

```diff
@@ -0,0 +1,319 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
@Kiciuk Please provide your name and email for author name & sign off in the commit message

https://www.kernel.org/doc/html/latest/process/submitting-patches.html#sign-your-work-the-developer-s-certificate-of-origin